toonz/sources/tnztools/tooloptions.cpp

```diff
@@ -171,7 +171,11 @@ ToolOptionControlBuilder::ToolOptionControlBuilder(ToolOptionsBox *panel,
 //-----------------------------------------------------------------------------
 
 QLabel *ToolOptionControlBuilder::addLabel(TProperty *p) {
-  QLabel *label = new QLabel(p->getQStringName());
+  QLabel *label;
+  if (p->getName() == "Sensitivity:")
+    label = new QLabel("");
```

Comment:
This line seems to be added to remove unwanted label for the snap sensitivity control.
How about setting the empty QString to `TProperty::m_qstringName` instead, by calling
` m_snapSensitivity.setQStringName("");` at the line 1014 in `BrushTool::updateTranslation()`?

Moreover, IMO it would better not to create `QLabel` here if `p->getQStringName().isEmpty()`. Because inserting the empty label causes uneven spacing between controls.